src/jsc/jsc_class.hpp

```diff
@@ -624,16 +622,25 @@ typename ClassType::Internal* ObjectWrap<ClassType>::get_internal(JSContextRef c
 }
 
 template <typename ClassType>
-void ObjectWrap<ClassType>::set_internal_property(JSContextRef ctx, JSObjectRef& instance,
-                                                  typename ClassType::Internal* internal)
+void ObjectWrap<ClassType>::set_internal(JSContextRef ctx, JSObjectRef& instance,
```

Comment:
What's the reason for renaming this function?
Also, given that it's been significantly reworked, a Doxygen header would be appreciated.

Reply:
I've added some documentation.

Reply:
The reason I've renamed it is because now I need it exposed externally (it was only used internally to the class before). The matching getter was named `get_internal` and I though the "property" part wasn't adding much clarity anyway.